Sources/DatadogObjc/DatadogConfiguration+objc.swift

```diff
@@ -23,6 +23,9 @@ public class DDEndpoint: NSObject {
     @objc
     public static func us3() -> DDEndpoint { .init(sdkEndpoint: .us3) }
 
+    @objc
+    public static func us5() -> DDEndpoint { .init(sdkEndpoint: .us5) }
+
```

Comment:
This needs adding coverage in [`DDConfiguration+apiTests.m`](https://github.com/DataDog/dd-sdk-ios/blob/4643224f4d2c685651e0abd9db1b74569a6855a6/Tests/DatadogTests/DatadogObjc/ObjcAPITests/DDConfiguration%2BapiTests.m#L48-L57) - to ensure this API can be called from Objective-C.

Reply:
Done, can you merge it for me if the update is ok, please?